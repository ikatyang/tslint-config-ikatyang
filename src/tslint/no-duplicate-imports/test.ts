// good

import {dirname, basename} from 'path';

// bad

import {readFileSync} from 'fs';
import {writeFileSync} from 'fs';
