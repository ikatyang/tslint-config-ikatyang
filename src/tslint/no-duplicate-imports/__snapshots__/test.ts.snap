// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`lint 1`] = `
// good

import {dirname, basename} from 'path';

// bad

import {readFileSync} from 'fs';
import {writeFileSync} from 'fs';
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ [Multiple imports from 'fs' can be combined into one.]

`;
